scripts/SkipOnWindows.js

```diff
@@ -11,11 +11,6 @@
 
 const SkipOnWindows = {
   suite() {
-    if (process.platform === 'win32') {
```

Comment:
Oops?

Reply:
nah, I wanna see all failures 🙂 Gonna add it back later (unless it's dead simple to support windows all over, which I doubt)